table_to_graph_importer/table_to_graph_importer.ipynb

```diff
@@ -0,0 +1,393 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b50a0dba",
+   "metadata": {},
+   "source": [
+    "# Importing Table Data into a Graph Database\n",
+    "\n",
+    "Through this short tutorial, you will learn how to import table data from files stored in local or online storage to a Memgraph graph database.\n",
+    "\n",
+    "If at any point you experience problems with this tutorial or something is\n",
+    "unclear to you, reach out on our [Discord server](https://discord.gg/memgraph).\n",
+    "\n",
+    "## 1. Prerequisites\n",
+    "\n",
+    "For this tutorial, you will need to install:\n",
+    "\n",
+    "- [Jupyter](https://jupyter.org/install): Jupyter is needed if you want to run this tutorial as a notebook.\n",
+    "- [Docker](https://docs.docker.com/get-docker/): Docker is used because Memgraph is a native Linux application and cannot be installed on Windows and macOS.\n",
+    "- [GQLAlchemy](https://pypi.org/project/gqlalchemy/): A Python OGM (Object Graph Mapper) that connects to Memgraph"
```

Comment:
I'd need some motivation here. Why are we doing this? We are importing tabular data to graph because we want to ease migrations to graph databases, etc. Most table data are stored in rows that could be represented as SQL. At least a few sentences here to get them a sense whether they'd like to read this article or not.